device/transport/mqtt/test/_mqtt_twin_receiver_test.js

```diff
@@ -6,7 +6,7 @@
 var assert = require('chai').assert;
 
 var MqttTwinReceiver = require('../lib/mqtt_twin_receiver.js').MqttTwinReceiver;
-var MqttProvider = require('./_fake_mqtt.js');
+var MqttProvider = require('../../../../common/transport/mqtt/test/_fake_mqtt.js');
```

Comment:
>../../../../common/transport/mqtt/test/ [](start = 28, length = 39)

ugh that really isn't great

Reply:
I think i'd rather even duplicate the code. those fakes are old patterns we should do away with. Please add a PBI to take care of that and lets tackle it during the milestone. technical debt at its worst here.

---
In reply to: [142813241](https://github.com/Azure/azure-iot-sdk-node/pull/101#discussion_r142813241) [](ancestors = 142813241)

Reply:
I agree.  This ../ pattern was in device/transport/mqtt before the merge, so I didn't worry too much about removing it.  I like the idea of getting rid of this entirely.
PBI opened.

---
In reply to: [142813528](https://github.com/Azure/azure-iot-sdk-node/pull/101#discussion_r142813528) [](ancestors = 142813528,142813241)